src/coreclr/vm/dllimport.h

```diff
@@ -150,15 +150,15 @@ enum NDirectStubFlags
 #ifdef FEATURE_COMINTEROP
     NDIRECTSTUB_FL_FIELDGETTER              = 0x00002000, // COM->CLR field getter
     NDIRECTSTUB_FL_FIELDSETTER              = 0x00004000, // COM->CLR field setter
-    // unused                               = 0x00008000,
-    // unused                               = 0x00010000,
-    // unused                               = 0x00020000,
+#endif // FEATURE_COMINTEROP
+    NDIRECTSTUB_FL_SUPPRESSGCTRANSITION     = 0x00008000,
```

Comment:
Side-note: This flag is likely going to be part of the fix for #46184

Reply:
I'll keep this one in the public section then and move the rest to internal.